cmd/aks-periscope/aks-periscope.go

```diff
@@ -35,19 +35,24 @@ func main() {
 	kubeletCmdCollector := collector.NewKubeletCmdCollector(exporter)
 	systemPerfCollector := collector.NewSystemPerfCollector(exporter)
 	helmCollector := collector.NewHelmCollector(exporter)
+	osmCollector := collector.NewOsmCollector(exporter)
+
+	collectors = append(collectors, containerLogsCollector)
+	collectors = append(collectors, dnsCollector)
+	collectors = append(collectors, kubeObjectsCollector)
+	collectors = append(collectors, networkOutboundCollector)
+
+	connectedClusterEnabled := false
+	for _, flag := range collectorFlags {
```

Comment:
Should it rather be a switch case instead of a for?

Reply:
Changing the implementation similar to Tatsat's suggestion, so I don't think a switch case will be needed